ONSdigital/eq-author
# eq-419 New editing canvas

### What is the context of this PR?

https://trello.com/c/nRSQZFAQ/419-new-editing-canvas-13

Implements new editing canvas design:

- new tab interface for switching between 'design'/'routing'
- updated design of page furniture (background, buttons, etc)
- updated page transitions
- updated answer adding/removing transitions

**Note: Does not implement new design of fields** 

### How to review 

- check navigation between sections/pages, etc
- check adding answers
- check moving page
- check deleting page
- check deleting sections
